tach/cli.py

```diff
@@ -207,15 +207,23 @@ def parse_arguments(
 ) -> tuple[argparse.Namespace, argparse.ArgumentParser]:
     parser = build_parser()
     parsed_args = parser.parse_args(args)
+
+    if len(args) < 1 or args[0] not in ["pkg", "clean", "sync"]:
+        # TODO: unify project config handling
+        fs.validate_project_config_path()
+
     return parsed_args, parser
 
 
 def tach_check(
+    exact: bool = False,
     root: str = ".",
```

Comment:
also due to `root` being added on main, just a nit
```suggestion
    root: str = ".",
    exact: bool = False,
```